src/test/java/ControllerTest.java

```diff
@@ -0,0 +1,21 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import controller.LottoController;
+import domain.Lottos;
+import domain.Price;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class ControllerTest {
+
+    @DisplayName("로또 구매 테스트")
+    @Test
+    public void testBuyLottos() {
+        LottoController testController = new LottoController();
+        Price testPrice = Price.from(3000);
+        Lottos testLottos = testController.buyLottos(testPrice);
+
+        assertThat(testLottos.getLottos().size()).isEqualTo(3);
+    }
```

Comment:
여기는 `given - when - then` 검색해서 적용해보면 좋아보여요~
```suggestion
    public void testBuyLottos() {
        // given
        LottoController testController = new LottoController();
        
        // when
        Price testPrice = Price.from(3000);
        Lottos testLottos = testController.buyLottos(testPrice);

        // then
        assertThat(testLottos.getLottos().size()).isEqualTo(3);
    }
```

그리고 테스트 코드는 결국엔 then 부분이 중요하니까 아래처럼 `actual`, `expected` 값까지 추가하면 다른 사람이 코드를 볼 경우 더 쉽게 파악이 가능해요 (나중에 역방향으로 추적하기 쉬움)

```java
    public void testBuyLottos() {
        // given
        LottoController testController = new LottoController();
        int expected = 3;
        
        // when
        Price testPrice = Price.from(3000);
        Lottos testLottos = testController.buyLottos(testPrice);
        int actual = testLottos.getLottos().size();

        // then
        assertThat(actual.isEqualTo(expected);
    }
```

Reply:
테스트 코드를 작성할 떄 이 패턴을 작성 하면 되는군요! 확실히 변수로 actual 값과 expected 값을 명시해 주니 훨씬 깔끔하고 알아보기 쉽네요! 여기서는 하나의 값을 테스트 하는데 사용되었는데 많은 값을 테스트 해야 할 경우에도 적용할 방법이 있을까요?

Reply:
저 같은 경우에는 그냥 뒤에 숫자 붙이고 있어요 ㅋㅋ
`expected1`, `expected2` 이것처럼요